README.md

```
@@ -10,93 +10,109 @@
-  </tr>
-  <tr>
-    <td width="85%">⚡ <a href="https://github.com/surajmandalcell/elec3d">elec3d</a> - Client Project</td>
-    <td><a href="http://elec3d.in/">elec3d.in</a></td>
-    <td width="20%" align="right"><img src="https://img.shields.io/badge/Private-gray?style=flat" /></td>
-  </tr>
-</table>
-</tr>
+#### Current Projects
+
+<p>
+  <img align="right" src="https://img.shields.io/github/stars/surajmandalcell/switch?style=flat&color=gray&labelColor=gray" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)